src/verify/hooks.ts

```diff
@@ -113,7 +106,6 @@ export function useVerifyPhoneNumber(phoneNumber: string, countryCallingCode: st
           clientPlatform: Platform.OS,
           clientVersion: DeviceInfo.getVersion(),
           clientBundleId: DeviceInfo.getBundleId(),
-          publicDataEncryptionKey: compressedPubKey(hexToBuffer(privateDataEncryptionKey)),
```

Comment:
Note that this change requires us to update the `verifyPhoneNumber` endpoint in `identity-service`, since the public DEK is currently a required parameter. AFAICT, it's not used for anything there, we just log it in the DB as an extra detail, so I think this is fine.

Reply:
See PR on `identity-service` [here](https://github.com/valora-inc/identity-service/pull/556/files).

Reply:
Thanks!